python/lsst/sims/GalSimInterface/galSimCatalogs.py

```diff
@@ -10,10 +10,12 @@
 GalSimStars
 """
 
+from builtins import zip
+from builtins import str
 import numpy as np
 import os
 import copy
-from itertools import izip
+
```

Comment:
I think I'm going to leave it so that there is a visual break between the `lsst` imports and the non-`lsst` imports.